avatar/alert.css

```diff
@@ -0,0 +1,129 @@
+@import url(../styles.css);
+
+html {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: var(--bg-color);
```

Comment:
instead of bg-color color name would be an appropriate option eg-(--bg-green)

Reply:
I have made the changes, could you please review them